lcls_tools/common/measurements/wire_scan.py

```diff
@@ -3,6 +3,8 @@
 from lcls_tools.common.devices.reader import create_lblm
 from lcls_tools.common.data.fit.projection import ProjectionFit
 from lcls_tools.common.measurements.measurement import Measurement
+from lcls_tools.common.measurements.tmit_loss import TMITLoss
+from lcls_tools.common.measurements.scan_utils.buffer_reservation import reserve_buffer
```

Comment:
Removing the scan_utils folder would make this import easier, etc. I think it's fine to have a buffer file at the top of measurement.